src/main/java/com/wire/kalium/helium/LoginClient.kt

```diff
@@ -198,7 +200,7 @@ open class LoginClient(client: Client) {
             return response.readEntity(NotificationList::class.java)
         } else if (status == 401) {   //todo nginx returns text/html for 401. Cannot deserialize as json
             response.readEntity(String::class.java)
-            throw AuthException(status)
+            throw HttpException(message = null, code = status, label = "ohh")
```

Comment:
the AuthException class was in xenon and got deletedbecouse is not used over there
i maby now it make sence to add it agian